src/software/backend/input/network/network_filter.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
```

Comment:
:exclamation: Copied from software/backend/input/network/networking with some refactoring and new functions